### ISSUE TITLE
Have a way to create packages for Archivematica from existing born-digital content

### ISSUE THREAD

**Issue by alexwlchan** [wellcometrust/platform#4065]
- [ ] Uncatalogued
- [x] Catalogued collections
- [ ] V drive

**Comment by alexwlchan** [comment]
Trudy sent me the spreadsheet of the 65 catalogued collections they want to export, and I've been able to grab all the files from Preservica for those collections, bar four. (I've downloaded them using the Preservica API, but I haven't compared to Preservica's checksums.)

Four files from the Dr. Dave King archive threw 500 errors when I tried to download them:

```
8d3477db-12a9-460f-afc3-283727bdc9eb PPKIN/2042/4/1/5
2308c98b-e8ac-4e40-b6d0-3d59d0755dfa PPKIN/2042/5/1/1/5
15bb086c-3d89-4e5e-bb14-202c848e958b PPKIN/5/1/3/5
705c0d12-b986-486b-8149-ec93e3088209 PPKIN/2042/7/1/6
```

Possibly significant: all four of these are an `index.html` page.

I haven't assembled Archivematica packages for these yet, but I do have the `metadata.csv`, so I think it's just zipping them up and uploading them to S3. 247GB in total.
